extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isExperimentOn} from '../../../src/experiments';
+import {user} from '../../../src/log';
+
+const TAG = 'amp-viewer-integration';
+
+
+export class AmpViewerIntegration extends AMP.BaseElement {
+
+  /** @override */
+  buildCallback() {
+    user().assert(isExperimentOn(this.win, TAG),
```

Comment:
Since this will be controlled by cache - the experiment is optional. Originally I thought we'd put this in a core. But since it's out of core, this flag is the same as cache inserting or not inserting the script. So, I suggest we remove the experiment.